examples/size.rs

```diff
@@ -31,10 +31,11 @@ fn main() {
         println!("      State.backtrace: {}", size_of_val(&state.backtrace));
     }
     #[cfg(not(feature = "backtrace"))]
-    {
+    fn size_of_state() {
         let state = error_chain::State {
             next_error: None,
         };
         println!("      State.next_error: {}", size_of_val(&state.next_error));
     }
+    size_of_state();
```

Comment:
Sure. But this isn't intended to merge, anyway, since it doesn't work completely.  #103 works, this just archives the attempt to make tests and examples work.